generators/app/index.js

```diff
@@ -195,45 +212,16 @@ module.exports = class extends Generator {
         }
         this.template('_README.md', 'README.md');
 
-        if (this.blueprintSubs.includes('client')) {
-            // copy files for the client generator
-            mkdirp('generators/client/templates');
-            this.template('generators/client/_index.js', 'generators/client/index.js');
-            this.template('generators/client/_files.js', 'generators/client/files.js');
-            this.template('generators/client/_prompts.js', 'generators/client/prompts.js');
-            this.template('generators/client/templates/_dummy.txt', 'generators/client/templates/_dummy.txt');
-        }
-        if (this.blueprintSubs.includes('server')) {
-            // copy files for the server generator
-            this.template('generators/server/_index.js', 'generators/server/index.js');
-        }
-        if (this.blueprintSubs.includes('entity')) {
-            // copy files for the entity generator
-            this.template('generators/entity/_index.js', 'generators/entity/index.js');
-        }
-        if (this.blueprintSubs.includes('entity-client')) {
-            // copy files for the entity-client generator
-            this.template('generators/entity-client/_index.js', 'generators/entity-client/index.js');
-        }
-        if (this.blueprintSubs.includes('entity-server')) {
-            // copy files for the entity-server generator
-            this.template('generators/entity-server/_index.js', 'generators/entity-server/index.js');
-        }
-        if (this.blueprintSubs.includes('entity-i18n')) {
-            // copy files for the entity-i18n generator
-            this.template('generators/entity-i18n/_index.js', 'generators/entity-i18n/index.js');
-        }
-        if (this.blueprintSubs.includes('languages')) {
-            this.template('generators/languages/_index.js', 'generators/languages/index.js');
-        }
-        if (this.blueprintSubs.includes('spring-controller')) {
-            // copy files for the spring-controller generator
-            this.template('generators/spring-controller/_index.js', 'generators/spring-controller/index.js');
-        }
-        if (this.blueprintSubs.includes('spring-service')) {
-            // copy files for the spring-service generator
-            this.template('generators/spring-service/_index.js', 'generators/spring-service/index.js');
-        }
+        this.blueprintSubs.forEach(generator => {
```

Comment:
You can include in this loop the handling of `test/_subgen.spec.ejs` and the `.yo-rc.json` : 
```
if (!generator.startsWith('entity-')) {
    this.template('test/_subgen.spec.ejs', `test/${generator}.spec.js`);
}
if (['entity', 'languages', 'spring-controller', 'spring-service'].includes(generator))) {
    this.template('test/templates/ngx-blueprint/.yo-rc.json.ejs', 'test/templates/ngx-blueprint/.yo-rc.json');
}
```

Reply:
done ;)